README.md

```diff
@@ -92,3 +92,47 @@ You need the container name or id of the healthchecks instance. You can get it b
 docker exec -it CONTAINER_NAME healthchecks_create_superuser.sh
 ```
 Follow the assistant that will show up to create a healthchecks superuser.
+
+### Docker-Compose
+Example docker-compose.yml
+```
+version: '3'
+
+services:
+  hc:
+    image: galexrt/healthchecks:latest
+    restart: always
+    ports:
+      - "8000:8000"
+      - "2525:2525"
+    volumes:
+      - HC_Data:/healthchecks
+      - HC_SQLite:/data
+    environment:
+      HC_HOST: "0.0.0.0"
+      HC_SECRET_KEY: "blablabla"
+      HC_ALLOWED_HOSTS: "hc.example.com"
```

Comment:
What if someone wants mutiple hosts here?

With the current system you would simply specify `HC_ALLOWED_HOSTS: '["*", "myotherhost", "example.com"]'` ? Then line 80 of the `entrypoint.sh` would take effect and put the value as is (plain) inside the config.